src/test/java/com/solvd/micro9/users/messaging/UserProducerIT.java

```diff
@@ -0,0 +1,49 @@
+package com.solvd.micro9.users.messaging;
```

Comment:
@eyakauleva integration tests should be located in dir, different from unit tests are

Reply:
@h1alexbel so they are. unit tests will be located in com.solvd.micro9.users.service

Reply:
maybe mark the dir like `integration` or `it`

WDYT?